jdaviz/core/freezable_state.py

```diff
@@ -151,11 +151,15 @@ def _set_zoom_radius_center(self, *args):
         if self.linked_by_wcs:
             image, i_ref = get_reference_image_data(self._viewer.jdaviz_app, self._viewer.reference)
             ref_wcs = image.coords
-            cr = ref_wcs.world_to_pixel_values((self.zoom_center_x, self.zoom_center_x+abs(self.zoom_radius)),  # noqa
-                                               (self.zoom_center_y, self.zoom_center_y))
+            if ( self.zoom_center_y+abs(self.zoom_radius) < 0):
```

Comment:
Nitpick: 
```suggestion
            if self.zoom_center_y + abs(self.zoom_radius) < 0:
```

I'm curious why this is the condition. I don't know why we're adding `zoom_radius`, and I expected `zoom_radius` would always be positive.

Based on your PR description, it seems like the condition could be
```python
            if self.zoom_center_y < 0:
```

Reply:
you are correct! That was my intention, I missed this when reviewing it myself

Reply:
Sweet. Could you add a comment on the line above, explaining the rationale?